.codecov.yml

```diff
@@ -2,32 +2,32 @@
 #  make codecov-validate
 
 codecov:
-  require_ci_to_pass: false
+  require_ci_to_pass: true
   notify:
     wait_for_ci: true
 
 coverage:
   status:
     project:
-      default:
-        informational: true
       unit-tests:
         target: auto
+        threshold: 1
         flags:
           - unit
       functional-tests:
+        threshold: 0.1
         target: auto
         flags:
           - functional
     patch:
       default:
-        informational: true
+        target: 50
```

Comment:
I'm sure much has been discussed about the targets for coverage @igor-sirotin. Is there any important context to share about why 50 is chosen for PRs? Is the goal to gradually increase `target` and disable `informational`?

Reply:
@ilmotta When we set up Codeclimate, we chose 50%, as it was the default value, and we had not better experience to set something higher/lower 🤷 

Yes, the plan is to gradually increase the target value. But for now the goal is to replace Codeclimate with Codecov.
Though now this 50% will also include `functional-tests` coverage. 

I guess we can consider increasing the number when we improve the testing framework so it would be much simpler to actually implement tests. Or we periodically analyse PRs and check what's the average `diff` coverage and increase accordingly.

Reply:
I'm keeping `informational: true` for `patch/unit-tests` and `patch/functional-tests` (just to show the numbers in status checks), because we currently don't force any specific tests to be implemented: just any tests to cover the changes. And that is checked with `patch { informational: false }`.

Reply:
It all makes sense to me as well :) Thanks for the explanation